### DISCUSSION TITLE
[feature] don't use reddit

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#5655] [Ideas]
No, you will not force me to use reddit. You could use lemmy. It's like reddit, but in the spirit of mastodon. I suppose that's a kind of a feature request. You should seriously think about avoiding reddit, problems don't come in isolation, probably more bad stuff are coming.

**Comment by vitoreiji** [comment]
You don't have to use reddit, you can use github discussions.

**Reply by ghost** [reply]
mention that in the explanations of when you open an issue.

**Reply by nicobao** [reply]
i upvote this, as i initially wanted to create an issue, which redirected me to create a discussion on reddit, which was closed, redirecting me to github, then i created an issue on github, which was moved to discussions

**Comment by dermitdemdino** [comment]
Tuta is rightly proud to use, promote and support open source wherever possible. I therefore find the idea of putting the Reddit forum on Lemmy very appealing.

**Comment by charlag** [comment]
Is Lemmy already actually usable? Last time I've seen it it was more of an alpha version
